src/olympia/devhub/templates/devhub/new-landing/components/footer.html

```diff
@@ -7,8 +7,9 @@ <h3 class="DevHub-Footer-sections-header">{{ _('Resources') }}</h3>
         <h4>{{ _('Documentation') }}</h4>
         <ul>
           <li><a href="https://developer.mozilla.org/Add-ons">{{ _('Add-ons') }}</a></li>
-          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('WebExtensions') }}</a></li>
+          <li><a href="https://developer.mozilla.org/Add-ons/WebExtensions">{{ _('Extensions') }}</a></li>
           <li><a href="https://developer.mozilla.org/Add-ons/Themes/Background">{{ _('Themes') }}</a></li>
```

Comment:
MDN doesn't always have a translation for a given locale, if you set the wrong locale you'll get a 404 (see https://developer.mozilla.org/eu/Add-ons/WebExtensions) rather than falling back to english. There's a bug filed for that but it remains unfixed. 

It's better to leave out the locale for MDN links and let MDN do the lookup.